Disable incompatible tests

online - disable online tests.

blame::simple::trivial_libgit2 - requires the libgit2 directory to be
a repository

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6bd25aa..f44190e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -349,7 +349,7 @@ IF (BUILD_CLAR)
 	ENDIF ()
 
 	ENABLE_TESTING()
-	ADD_TEST(libgit2_clar libgit2_clar -ionline)
+	ADD_TEST(libgit2_clar libgit2_clar -xonline -xblame::simple::trivial_libgit2 -xnetwork::remote::remotes::single_branch -xnetwork::remote::remotes::restricted_refspecs)
 ENDIF ()
 
 IF (TAGS)
